client/main/backstage.js

```diff
@@ -700,12 +698,31 @@ var BackstageModel = Backbone.Model.extend({
             this.set('activated', false);
          });
     },
-    requestSave: function(path) {
-        this.instance.save(path)
+    requestSave: function(path, overwrite) {
+
+        if (overwrite === undefined && typeof path === "boolean") {
```

Comment:
oh right, you've explained it in slack, do you want to add a comment explaining it here too?